lib/ansible/modules/messaging/rabbitmq_policy.py

```diff
@@ -149,6 +168,7 @@ def main():
         tags=dict(type='dict', required=True),
         priority=dict(default='0'),
         node=dict(default='rabbit'),
+        extra_ctl_paths=dict(default=list(), type='list'),
```

Comment:
i would move common arguments to a shared argspec (to be merged with the moduel specific) hosted in module_utils